modules/instruments/test/TestPlan.md

```diff
@@ -0,0 +1,19 @@
+# Instrument Test Plan
+
+1. Verify that in order to access the module, the user must either have the `Across all sites access
+candidate profiles` permission, or have the 'Edit Candidate Parameters' permission and be at the same
+site as the visit, or at the same site as one of the candidate's other visits.
+2. Select and instrument in the list of the session instruments.
```

Comment:
Add case above to test custom instrument permissions defined in the config.xml file (see `instrumentPermissions` tag)

this is very straight forward, if in the config.xml file you add a test_name and a permission name of your choosing, you should only be able to access that instrument if you have said permission. if the permission is not in the existing loris permissions, you would have to add it to the permissions table